src/main/java/appeng/parts/PartPlacement.java

```diff
@@ -429,15 +387,6 @@ public static IFacadePart isFacade( final ItemStack held, final AEPartLocation s
 			return ( (IFacadeItem) held.getItem() ).createPartFromItemStack( held, side );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method appeng.parts.PartPlacement.isFacade(ItemStack, AEPartLocation) appears to call the same method on the same object redundantly [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=fb-contrib%3APRMC_POSSIBLY_REDUNDANT_METHOD_CALLS)